test/utils/mqtt.spec.js

```diff
@@ -0,0 +1,91 @@
+const EventsEmitter = require('events');
+const chai = require('chai');
+const sinon = require('sinon');
+
+const { expect } = chai;
+const { Mqtt } = require('./../../utils');
+
+chai.use(require('sinon-chai'));
+
+class Device extends EventsEmitter {};
```

Comment:
```suggestion
class Device extends EventsEmitter {
   publish(topic, payload) => {
     device.emit('message', topic, payload);
   };
};
```

I think that by doing this you are able to test the `mqtt.sendMessage` method, which is not tested right now. What do you think about that? 🙂

Maybe even create a DeviceFactory?

Reply:
Greate idea! Added `publish` method, and tested `sendMessage`. 
But I don't see a point for now to create DeviceFactory.